account_mtd/models/mtd_api_request_tracker.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api
+
+class MtdApiRequestTracker(models.Model):
+    _name = 'mtd.api_request_tracker'
+    _description = "authorisation request table"
+    
+    user_id = fields.Integer('User Id', required=True)
+    api_id = fields.Integer('API Id', required=True)
```

Comment:
This one can stay however (as it would read `Api Id` if removed). Would like it to read `api_id = fields.Integer(string='API Id', required=True)` though